### PR TITLE
Remove eslint-disable lines

### PR DESCRIPTION
**Pull request by Jephuff**

### CODE REVIEW

File: src/packages/@ncigdc/components/FacetSelection.js [hunk 1]
```diff
@@ -1,118 +1,110 @@
 /* @flow */
-/* eslint-disable */
 
-import React from "react";
-import _ from "lodash";
-import { css } from "glamor";
+import React from 'react';
+import _ from 'lodash';
+import { css } from 'glamor';
```
Comment (shanewilson) [inline src/packages/@ncigdc/components/FacetSelection.js]: shouldn't these be `"`?

Comment (shanewilson) [inline src/packages/@ncigdc/components/FacetSelection.js]: oh wait we went the other way - so was this file missed? I don't get how files are missed when running `eslint . --fix`

Comment (shanewilson) [inline src/packages/@ncigdc/components/FacetSelection.js]: oh again - I am dumb I get it

Comment (Jephuff) [inline src/packages/@ncigdc/components/FacetSelection.js]: 👍

**Pull request merged**